pkg/awsManager/ec2Manager.go

```diff
@@ -2,25 +2,25 @@ package awsManager
 
 import (
 	"errors"
-	"fmt"
+	"strings"
+
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/go-logr/logr"
 	clientpkg "github.com/openshift/aws-account-shredder/pkg/aws"
-	"strings"
 )
 
-// this lists all the instances that are eligible for deletion based on the tags and stored them in instances to be deleted
+// ListEc2InstancesForDeletion this lists all the instances that are eligible for deletion based on the tags and stored them in instances to be deleted
 // this only creates an array of pointers and does not delete the instances
-func ListEc2InstancesForDeletion(client clientpkg.Client) []*string {
+func ListEc2InstancesForDeletion(client clientpkg.Client, logger logr.Logger) []*string {
```

Comment:
Why was this unexported?

Reply:
It wasn't, just added logger as a parameter

Reply:
probably should be becuase it isn't and shouldn't be used anywhere outside this file